src/pages/resources/resources.ts

```diff
@@ -630,10 +630,13 @@ export class ResourcesPage implements OnInit, AfterViewInit {
 
   ionViewDidLoad() {
     this.appGlobalService.generateConfigInteractEvent(PageId.LIBRARY, this.isOnBoardingCardCompleted);
+
+    this.events.subscribe('tab.change', () => {
```

Comment:
There is another events subscription for tab.change in line number 202. You can add there.